### PR TITLE
more information in responses when user is viewing own profile

### PR DESCRIPTION
**Pull request by 2is10**
@eishay @ahsu1230 

I’ll resolve the conflicts before merging.

### CODE REVIEW

**Comment by 2is10** [comment]
When a user is viewing his/her own profile, clients need to know which libraries are private and which are system libraries. Currently, the differences between `LibraryCardInfo` and `OwnLibraryCardInfo` are:

```
- owner
- caption
+ kind
+ visibility
```

File: kifi-backend/modules/shoebox/app/com/keepit/model/LibraryView.scala [hunk 1]
```diff
@@ -98,12 +126,9 @@ case class LibraryCardInfo(id: PublicId[Library],
   numFollowers: Int,
   followers: Seq[BasicUser],
   caption: Option[String])
+    extends BaseLibraryCardInfo(id, name, description, color, image, slug, numKeeps, numFollowers, followers)
 
 object LibraryCardInfo {
-  val writesWithoutOwner = Writes[LibraryCardInfo] { o => // for case when receiving end already knows the owner
-    JsObject((Json.toJson(o).asInstanceOf[JsObject].value - "owner").toSeq)
-  }
-
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/model/LibraryView.scala:106]: happy to see this go, felt kind of hacky

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala [hunk 2]
```diff
@@ -1257,78 +1257,91 @@ class LibraryCommander @Inject() (
...
       lib => // may want to optimize queries below into bulk queries
         val image = ProcessedImageSize.pickBestImage(idealSize, libraryImageRepo.getForLibraryId(lib.id.get))
         val numKeeps = keepRepo.getCountByLibrary(lib.id.get)
         val (numFollowers, followersSample) = if (lib.memberCount > 1) {
           val count = libraryMembershipRepo.countWithLibraryIdAndAccess(lib.id.get, LibraryAccess.READ_ONLY)
-          val sample = libraryMembershipRepo.pageWithLibraryIdAndAccess(lib.id.get, 0, 2, Set(LibraryAccess.READ_ONLY)) map {
-            lm =>
-              basicUserRepo.load(lm.userId)
+          val sample = libraryMembershipRepo.pageWithLibraryIdAndAccess(lib.id.get, 0, 2, Set(LibraryAccess.READ_ONLY)) map { lm =>
+            basicUserRepo.load(lm.userId)
```
Comment (eishay) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala]: i merged something that's a bit more optimized

Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala]: nice! conflicts resolved

**Pull request merged**